### DISCUSSION TITLE
Ingestion Settings Tab is missing

### DISCUSSION THREAD

**Discussion by hopewise** [SigNoz/signoz#6094] [Q&A]
Hello

We have installed signoz **v0.54.0** on AWS EC2 server, but we noticed that ingestion settings tab is missing as per snap shot, as per this doc https://signoz.io/docs/ingestion/signoz-cloud/keys/

My user is with admin role. We installed signoz via the Install Script as this doc https://signoz.io/docs/install/docker/#install-signoz-using-the-install-script

<img width="1201" alt="Screenshot 2024-09-30 at 2 33 44 PM" src="https://github.com/user-attachments/assets/48843bbf-dbb1-4206-ab59-b6bbaf17f6c1">

**Comment by ckshen8** [comment]
Same here.  I deployed Signoz v0.55.0 on Ubuntu 24.04 using Docker Compose using the instruction referenced here:
https://signoz.io/docs/install/docker/#install-signoz-using-docker-compose
```
git clone -b main https://github.com/SigNoz/signoz.git && cd signoz/deploy/
docker compose -f docker/clickhouse-setup/docker-compose.yaml up -d
docker ps
```
And the ingestion tab is missing.

**Comment by amurilloc** [comment]
Converted to issue and apparently is a Cloud version-only feature:
https://github.com/SigNoz/signoz/issues/6131#issuecomment-2398601936

**Reply by ckshen8** [reply]
Thank you.  Very helpful @amurilloc
